decidim-accountability/lib/decidim/api/result_type.rb

```diff
@@ -27,6 +26,12 @@ class ResultType < Decidim::Api::Types::BaseObject
       field :parent, Decidim::Accountability::ResultType, "The parent result", null: true
       field :status, Decidim::Accountability::StatusType, "The status for this result", null: true
       field :timeline_entries, [Decidim::Accountability::TimelineEntryType, { null: true }], "The timeline entries for this result", null: true
+
+      # Unsure why this field is required but it is requested in the integration
```

Comment:
The ComponentInterface on the result has been added in the original PR #5584 
If the only place where the field is being used is in `integration_schema_spec.rb`, then we can safely patch and remove the request. 

The integration schema spec has been added by me when i have migrated from old to new syntax, aiming to preserve the functionality.